src/libdrakvuf/vmi.c

```diff
@@ -547,6 +547,8 @@ event_response_t int3_cb(vmi_instance_t vmi, vmi_event_t* event)
     trap_info.proc_data.pid       = proc_data.pid;
     trap_info.proc_data.ppid      = proc_data.ppid;
     trap_info.proc_data.userid    = proc_data.userid;
+    if (drakvuf_get_os_type(drakvuf) == VMI_OS_LINUX)
+        trap_info.proc_data.tid    = proc_data.tid;
```

Comment:
You can remove this

Reply:
I think this should be there because when the user specifies thread id for injection, I need to read the thread it when it returns.
here,
Pid = TGID in Linux
Tid = thread id in Linux

Reply:
Ok but in that case just add it to Windows as well so proc_data.tid gets filler there too

Reply:
Sure

Reply:
Windows compatibility?

Reply:
There should be no OS checks in this level, need to remove `if (drakvuf_get_os_type(drakvuf) == VMI_OS_LINUX)`